dashboard/test/models/peer_review_test.rb

```diff
@@ -192,6 +193,13 @@ def track_progress(level_source_id, user = @user)
             name: I18n.t('peer_review.review_in_progress'),
             result: ActivityConstants::UNSUBMITTED_RESULT,
             locked: false
+        },
+        {
+            status: 'not_started',
+            name: 'Review a new submission',
+            result: ActivityConstants::UNSUBMITTED_RESULT,
+            icon: '',
+            locked: false
         }
     ]
```

Comment:
Ah - the comment is wrong. We're expecting 3 elements in the summary. 2 reviews, and one link to a new review